docs/pages/cloud/faq.mdx

```diff
@@ -43,14 +51,15 @@ There is no need to open any ports on your infrastructure for inbound traffic.
 
 ## How can I access the tctl admin tool?
 
-We have made changes to allow you to log into your cluster using `tsh`, then use `tctl` remotely:
+Find the appropriate download at [Teleport Cloud Downloads](./downloads.mdx). Use the enterprise version of `tctl` and `tsh`.
```

Comment:
```suggestion
Find the appropriate download at [Teleport Cloud Downloads](./downloads.mdx). Use the Enterprise version of `tctl`.
```

My understanding is that there is no separate Enterprise version of `tsh` ([source](https://github.com/gravitational/teleport/pull/10920#discussion_r848898440)).